DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/BulkUpload/UploadFailed.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@inject IConfiguration Configuration
+@using Microsoft.Extensions.Configuration
+
+@{
+  ViewData["Title"] = "Failed - Bulk upload/update delegates";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">Bulk upload failed</h1>
+
+    <p class="nhsuk-body-m">
+      The file that you uploaded does not have the correct column headers on the first row.
+    </p>
+
+    <p class="nhsuk-body-m">Ensure that you have downloaded the template using "Download centre delegates for bulk upload" link before adding/amending delegates and uploading.</p>

```

Comment:
This line of text will also need updating to match the actual system (Download delegates button)